CommuniTEAM/CommuniTEA
# Dockerize Project

### In this PR, I'm adding/changing the following:

- Added a docker-compose.yml with React app and PostgreSQL database containers configured.
- Removed individual Dockerfile in favor of the docker-compose so that all containers can run as a unit.
- Enabled the hot reload module with Vite to work with the Dockerized setup.
- Changed app's README file to match the changes to the Docker setup.

### Please do the following to test my code:

1. Navigate to the project directory in your terminal
2. Create the database volume by running `docker volume create communitea-db`
3. Run `docker compose up`
4. Navigate to http://localhost:3000/ in your browser
5. See that the app is up and running!
6. Also, test hot reload by changing some text in App.tsx -- changes should show immediately on the page in your browser
